packages/hash/api/src/graph/index.ts

```diff
@@ -34,3 +40,16 @@ export const createGraphClient = (
 
   return new GraphApiClient(config, basePath, axiosInstance);
 };
+
+export const ensureHashAppIsInitialized = async (params: {
```

Comment:
Glad we have a shared method for this now.

I think we should work on the naming, perhaps something like `ensureSystemGraphIsInitialized` or `ensureSystemPrerequisitesAreInGraph` or something equally unpleasant but at least clear.

This does not initialize the HASH app, it just makes sure the requisite information is within the graph so that the app can continue with initializing

Reply:
I also prefer `ensureSystemGraphIsInitialized`, have renamed this in https://github.com/hashintel/hash/pull/1596/commits/2ee0ca0cfc695f172ec8d7969068556617debc0c